src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String [] data = new String[] {
+                "26.04.2019 John 4 50",
+                "05.04.2019 Andrew 3 200",
+                "10.04.2019 John 7 100",
+                "22.04.2019 Kate 9 100",
+                "25.06.2019 John 11 50",
+                "26.04.2019 Andrew 3 150",
+                "13.02.2019 John 7 100",
+                "26.04.2019 Kate 9 100"
+        };
+        String [] name = new String[] {
```

Comment:
The array `name` should be plural since it contains multiple names, consider renaming it to `employeeNames` to be more descriptive.